### PR TITLE
Fixing Export

### PR DESCRIPTION
**Pull request by bloodbare**
### Description
Describe the proposed changes made in this PR.

### How was this PR tested?
Describe how you tested this PR.

### CODE REVIEW

**Review by jotare — APPROVED** [review]

**Review by hermeGarcia — APPROVED** [review]

File: nucliadb_ingest/nucliadb_ingest/orm/resource.py [hunk 1]
```diff
@@ -244,7 +244,7 @@ async def generate_index_message(self) -> ResourceBrain:
                     brain.apply_field_vectors(field_key, vo, [], {})
         return brain
 
-    async def generate_broker_message(self) -> BrokerMessage:
+    async def generate_broker_message(self, cf: bool = True) -> BrokerMessage:
```
Comment (lferran) [inline nucliadb_ingest/nucliadb_ingest/orm/resource.py:247]: Does `cf` mean `cloud_file`? Why not naming the parameter `cloud_file` or to something more meaningful?

File: nucliadb_ingest/nucliadb_ingest/orm/resource.py [hunk 2]
```diff
@@ -263,27 +263,70 @@ async def generate_broker_message(self) -> BrokerMessage:
...
+                bm.conversations[field_id].CopyFrom(value)
+            elif type_id == FieldType.KEYWORDSET:
+                value = await field.get_value()
+                bm.keywordsets[field_id].CopyFrom(value)
+            elif type_id == FieldType.DATETIME:
+                value = await field.get_value()
+                bm.datetimes[field_id].CopyFrom(value)
+            elif type_id == FieldType.LAYOUT:
+                value = await field.get_value()
+                bm.layouts[field_id].CopyFrom(value)
```
Comment (lferran) [inline nucliadb_ingest/nucliadb_ingest/orm/resource.py:287]: What if the type is non of these? Do we need a `else: raise ValueError()` branch at the end?

**Review by lferran — APPROVED** [review]
I left a couple of comments, but LGTM!

File: nucliadb_ingest/nucliadb_ingest/orm/resource.py (hunk 1 above)
Comment (bloodbare) [reply nucliadb_ingest/nucliadb_ingest/orm/resource.py:247]: changing to `full` in order to make it more clear, it means all the bm, not the pointers

File: nucliadb_ingest/nucliadb_ingest/orm/resource.py (hunk 2 above)
Comment (bloodbare) [reply nucliadb_ingest/nucliadb_ingest/orm/resource.py:287]: in case there is else we do not want to raise, we do not want to export.

**Pull request merged**